rotkehlchen/assets/utils.py

```diff
@@ -411,3 +420,14 @@ def operator(self) -> Literal['IN', 'NOT IN']:
         if self == IgnoredAssetsHandling.EXCLUDE:
             return 'NOT IN'
         return 'IN'
+
+
+CHAIN_TO_WRAPPED_TOKEN = {
+    SupportedBlockchain.ETHEREUM: A_WETH,
+    SupportedBlockchain.ETHEREUM_BEACONCHAIN: A_WETH,
```

Comment:
This one doesn't have a wrapped token

Reply:
You resolved this one but it still there. Is it needed for some reason? Can you make it final also?